x-pack/legacy/plugins/siem/public/containers/case/use_get_cases.tsx

```diff
@@ -246,10 +247,17 @@ export const useGetCases = (): UseGetCases => {
     [state.filterOptions, state.queryParams]
   );
 
+  const refetchCases = useCallback(() => {
+    fetchCases(state.filterOptions, state.queryParams);
```

Comment:
Is this an `async` function?

Reply:
we do not want to await for this since we have state updates, so we contain the async in the one call. check this: https://dev.to/n1ru4l/homebrew-react-hooks-useasynceffect-or-how-to-handle-async-operations-with-useeffect-1fa8